src/test/resources/exportJson/all.json

```diff
@@ -0,0 +1,4 @@
+{"object_type":"node","id":0,"labels":["User"],"properties":{"name":"foo","age":"42","male":"true","kids":["a","b","c"]}}
```

Comment:
can we inline all these files? i.e. generate them from literal maps/collections + jackson?